YesGraph/yesgraph-superwidget
# Yahoo contacts

#### What’s this PR do?
- Adds Yahoo as a contact importer
- Separates out the stable code from the development code

#### Does this require changes on the API side?
Nope. The related backend code has already been deployed.

#### How should this be manually tested?
Go to https://www.yesgraph.com/demo?dev=1 This will load the Superwidget with the development code (not the stable code). You should be able to import Yahoo contacts.

#### What are the relevant tickets?
https://app.asana.com/0/search/132706980133933/111167391230934

#### Screenshots (if appropriate)
<img width="443" alt="screen shot 2016-05-12 at 11 18 09 am" src="https://cloud.githubusercontent.com/assets/10701968/15225425/3b47dbb4-1833-11e6-9f11-87b012d0625e.png">

#### Does the knowledge base need an update?
There's a Yahoo setup guide in the docs already.
